simde/arm/neon/qshrun_n.h

```diff
@@ -35,6 +36,16 @@ HEDLEY_DIAGNOSTIC_PUSH
 SIMDE_DISABLE_UNWANTED_DIAGNOSTICS
 SIMDE_BEGIN_DECLS_
 
+#if defined(SIMDE_ARM_NEON_A64V8_NATIVE)
+  #define simde_vqshrunh_n_s16(a, n) HEDLEY_STATIC_CAST(uint8_t, vqshrunh_n_s16((a), (n)))
```

Comment:
Why is this cast needed? `uint8_t` is the return type: https://developer.arm.com/architectures/instruction-sets/intrinsics/vqshrunh_n_s16

Reply:
When I build with cross-file `aarch64-clang-15-ccache.cross`, it will report a warning of implicit sign conversion. Maybe there is a better way to deal with this issue?
```
test/arm/neon/qshrun_n.cpp:69:18: error: implicit conversion changes signedness: 
'int8_t' (aka 'signed char') to 'uint8_t' (aka 'unsigned char') [-Werror,-Wsign-conversion]
                    uint8_t r1 = simde_vqshrunh_n_s16(a, 1);
                            ~~   ^~~~~~~~~~~~~~~~~~~~~~~~~~
../test/arm/neon/../../../simde/arm/neon/qshrun_n.h:40:38: note: expanded from macro 'simde_vqshrunh_n_s16'
  #define simde_vqshrunh_n_s16(a, n) vqshrunh_n_s16((a), (n))
```

Reply:
https://github.com/llvm/llvm-project/issues/71751